README.md

```diff
@@ -1,5 +1,15 @@
 # Introduction 
-Project Coimbra
+Project Coimbra 
+
+# Setup for development
+1. In root folder, run `git submodule init` to clone nested repo (DryWetMidi)
+2. Launch solution ProjectCoimbra.UWP/Project.Coimbra.sln
+3. Create your own certificate for development by  
+    3.1. In Visual Studio, Solution Explorer, open Project.Coimbra (Universal Windows) / Package.appxmanifest   

```

Comment:
I don't believe these lists are right still.

I believe it should be like the following, which worked for me when I tested it

```
3. Create your own certificate for development by  
    1. In Visual Studio, Solution Explorer, open Project.Coimbra (Universal Windows) / Package.appxmanifest
```

Reply:
I have changed to your advice. But using 3.1 does work, and this is my preference when having numbered list :-)